tests/genomic_ingest.json

```
@@ -0,0 +1,52 @@
+[
+    {
+        "program_id": "SYNTHETIC-2",
+        "genomic_id": "HG00096.cnv.vcf",
+        "main": {
+            "access_method": "s3://s3.us-east-1.amazonaws.com/1000genomes/release/20130502/ALL.chr22.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz?public=true",
+            "name": "HG00096.cnv.vcf.gz"
+        },
+        "index": {
+            "access_method": "s3://s3.us-east-1.amazonaws.com/1000genomes/release/20130502/ALL.chr22.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz?public=true",
+            "name": "HG00096.cnv.vcf.gz.tbi"
+        },
+        "metadata": {
+            "sequence_type": "wgs",
+            "data_type": "variant",
+            "reference": "hg37"
+        },
+        "samples": [
+            {
+                "genomic_sample_id": "HG00096",
+                "donor_sample_id": "SAMPLE_REGISTRATION_1"
```

If we change to submitter_sample_id we will need to edit the examples
```suggestion
                "submitter_sample_id": "SAMPLE_REGISTRATION_1"
```